solidity/test/helpers/Constants.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
         ROLE_OWNER: web3.utils.keccak256('ROLE_OWNER'),
         ROLE_GOVERNOR: web3.utils.keccak256('ROLE_GOVERNOR'),
         ROLE_MINTER: web3.utils.keccak256('ROLE_MINTER'),
-        ROLE_WHITELIST_ADMIN: web3.utils.keccak256('ROLE_WHITELIST_ADMIN')
+        ROLE_WHITELIST_ADMIN: web3.utils.keccak256('ROLE_WHITELIST_ADMIN'),
+        ROLE_SEEDER: web3.utils.keccak256('ROLE_SEEDER')
     },
 
     ETH_RESERVE_ADDRESS: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
Maybe it's time we added this one in some common helper file.

Reply:
This is the common helper file, no?

Reply:
Yes, sorry :)